test/extended/testdata/builds/simple-pipeline-bc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: minimalpipeline
+spec:
+  strategy:
+    jenkinsPipelineStrategy:
+      jenkinsfile: |-
+        node() {
+          sh 'exit 0'
```

Comment:
can't get any simpler :-)

Reply:
the latter - executing in primary jenkins pod ("computer" in jenkins parlance)

so we do not need any of the sidecar parlance